internal/datastore/postgres/watch.go

```diff
@@ -189,21 +213,19 @@ func (pgd *pgDatastore) loadChanges(ctx context.Context, revision xid8) (*datast
 			}
 		}
 
-		if createdXID.Uint == revision.Uint {
-			tracked.AddChange(ctx, postgresRevision{revision, noXmin}, nextTuple, core.RelationTupleUpdate_TOUCH)
-		} else if deletedXID.Uint == revision.Uint {
-			tracked.AddChange(ctx, postgresRevision{revision, noXmin}, nextTuple, core.RelationTupleUpdate_DELETE)
+		if _, found := filter[createdXID.Uint]; found {
+			tracked.AddChange(ctx, postgresRevision{createdXID, noXmin}, nextTuple, core.RelationTupleUpdate_TOUCH)
+		}
+		if _, found := filter[deletedXID.Uint]; found {
```

Comment:
is it possible for both these conditions to be true?

Reply:
Yes, if the revision range is large enough to encompass the tuple being both created and deleted.